lib/pages/login/login_page.dart

```diff
@@ -0,0 +1,103 @@
+                  crossAxisAlignment: CrossAxisAlignment.stretch,
+                  children: [
+                    Container(
+                      margin: const EdgeInsets.only(top: 150.0),
+                      child: SvgPicture.asset('assets/images/logo.svg'),
+                    ),
+                    const SizedBox(height: 100),
+                    TextFormField(
+                      keyboardType: TextInputType.emailAddress,
+                      controller: _loginController.emailController,
+                      decoration: InputDecoration(
+                          labelStyle: TextStyle(color: Colors.white30),
+                          floatingLabelBehavior: FloatingLabelBehavior.never,
+                          border: OutlineInputBorder(
+                              borderSide: BorderSide.none,
+                              borderRadius: BorderRadius.circular(12.0)),
+                          fillColor: Colors.white24,
+                          filled: true,
+                          labelText:
+                              AppLocalizations.of(context).titleGeneralEmail),
+                      style: TextStyle(color: Colors.white),
+                      validator: _emailValidator,
+                    ),
+                    const SizedBox(height: 20),
+                    TextFormField(
+                      keyboardType: TextInputType.text,
+                      controller: _loginController.passwordController,
+                      decoration: InputDecoration(
+                          labelStyle: TextStyle(color: Colors.white30),
+                          floatingLabelBehavior: FloatingLabelBehavior.never,
+                          border: OutlineInputBorder(
+                              borderSide: BorderSide.none,
+                              borderRadius: BorderRadius.circular(12.0)),
+                          fillColor: Colors.white24,
+                          filled: true,
+                          labelText: AppLocalizations.of(context)
+                              .titleGeneralPassword),
+                      obscureText: true,
+                      style: TextStyle(color: Colors.white),
+                    ),
+                    const SizedBox(height: 50),
+                    ElevatedButton(
+                      child: Text(AppLocalizations.of(context).buttonLogin),
+                      onPressed: () {
+                        _loginController.attemptLogin();
+                      },
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  String _emailValidator(String value) {
+    if (value == null || value.isEmpty) {
+      return AppLocalizations.of(Get.context).errorValidationEmailEmpty;
+    } else if (!EmailValidator.validate(value)) {
+      return AppLocalizations.of(Get.context).errorValidationEmailInvalid;
+    }
+    return null;
```

Comment:
I recommend following this convention from [ruby](https://nimblehq.co/compass/development/code-conventions/ruby/#conditional-expressions) 
```suggestion
    if (value == null || value.isEmpty) {
      return AppLocalizations.of(Get.context).errorValidationEmailEmpty;
    }
    
    if (!EmailValidator.validate(value)) {
      return AppLocalizations.of(Get.context).errorValidationEmailInvalid;
    }
    return null;
```